crates/ruff_python_parser/src/parser/tests/parser.rs

```diff
@@ -499,7 +499,7 @@ continue
 break
 
 del a
-del a, b, 1, 1 + 2,
+del a, b,
```

Comment:
The new parser now raises a syntax error for invalid delete target. This is also a hard syntax error in the CPython parser.

There could be an argument to make this a "soft" syntax error. I think I'd go through all of the hard syntax errors the new parser will raise and decide if it should actually be a "soft" syntax error.